### PR TITLE
feat(api): get etoro asset details

### PR DESCRIPTION
**Pull request by gierczakm**

### CODE REVIEW

File: apps/api/src/etoro/etoro.controller.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import { ApiOperation, ApiTags } from '@nestjs/swagger';
+
+import { EtoroService } from './etoro.service';
+
+@ApiTags('Etoro.com')
+@Controller('etoro')
+export class EtoroController {
+  constructor(private readonly etoroService: EtoroService) {}
+
+  @Get('assetData/:symbol')
```
Comment (mjastrzebowski) [inline apps/api/src/etoro/etoro.controller.ts:11]: urls are case insensitive, name it just `search` or `asset`?

**Review by mjastrzebowski — APPROVED** [review]

File: apps/api/src/etoro/etoro.service.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import { AxiosResponse } from 'axios';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+@Global()
+@Injectable()
+export class EtoroService {
+  constructor(private http: HttpService) {}
+
+  getAssetData(symbol: string): Observable<any> {
```
Comment (mjastrzebowski) [inline apps/api/src/etoro/etoro.service.ts:11]: Please add interface already (in `@postonk/types`) so you can use it later in the app too

Comment (gierczakm) [reply apps/api/src/etoro/etoro.service.ts:11]: no need as I'm returning JSON, I will need to parse it in app and then I will have to define inteface to assign type

Comment (mjastrzebowski) [reply apps/api/src/etoro/etoro.service.ts:11]: That's why I did all this setup, so we can define the responses and see it in documentation for easy use

**Pull request merged**